db/migrate/20150609191815_create_songs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSongs < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp'
+
+    create_table :songs, id: :uuid do |t|
+      t.timestamps null: false
+
+      t.string :title
+      t.string :artist
+      t.string :album
+      t.integer :duration
+    end
+  end
```

Comment:
Could think about a uniqueness constraint on some combination of these attributes, just so you don't get duplicate records.  Maybe `title, album`?